newsfragments/113.feature

```diff
@@ -0,0 +1 @@
+Add Intent Mapper and Metric wrapper features.
```

Comment:
Oh you should prolly add a newsfragment too.

Reply:
Yeap on my next commit.